extensions/amp-nested-menu/amp-nested-menu.md

```diff
@@ -0,0 +1,217 @@
+---
+$category@: layout
+formats:
+  - websites
+teaser:
+  text: Displays a drilldown menu with arbitrary levels of nested submenus.
+experimental: true
+---
+
+<!--
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# `amp-nested-menu`
+
+Drilldown menu component with items that open nested submenus on tap.
+
+<table>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td><div><a href="https://amp.dev/documentation/guides-and-tutorials/learn/experimental">Experimental</a>; activated by the <code>amp-nested-menu</code> experiment.</div></td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><div>None (<code>amp-sidebar</code> lazy loads the component)</div></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/control_layout">Supported Layouts</a></strong></td>
+    <td>fill</td>
+  </tr>
+</table>
+
+## Overview
+
+`<amp-nested-menu>` provides a way to organize and display multiple levels of menu content inside an [`<amp-sidebar>`](../amp-sidebar/0.1/amp-sidebar.md). The component can be used jointly with [`<amp-mega-menu>`](../amp-mega-menu/amp-mega-menu.md) to create a responsive menu.
+
+## Usage
+
+The `<amp-nested-menu>` component must be placed inside `<amp-sidebar>`. The component may contain the following AMP elements:
+
+- `<amp-img>`
+- `<amp-list>`
+- `<amp-accordion>`
+
+### Nested submenus
+
+`<amp-nested-menu>` supports nesting one or more layers of submenus. It uses the following attributes on its descendants as identifiers for the submenu functionality:
+
+- `amp-nested-submenu`: this identifies a submenu container. The element is initially out of view but, when opened, slides in and takes the place of the parent menu (either `<amp-nested-menu>` or another submenu).
```

Comment:
I'd update to 

`amp-nested-submenu`: this identifies a hidden submenu container.  The element slides in and takes the place of its parent menu when opened.